### ISSUE TITLE
Check Dockerfiles/BuildTest/Dockerfile is still used, otherwise delete it

### ISSUE THREAD

**Issue by wainersm** [kata-containers/tests#4222]
Recently I touched the file `Dockerfiles/BuildTest/Dockerfile` in https://github.com/kata-containers/tests/pull/4194 but actually I don't know if that dockerfile is still used.

The commit f9fde42170d7ca966eebb90e44341e7f12342279 that introduced it says:
```
commit f9fde42170d7ca966eebb90e44341e7f12342279
Author: Gabriela Cervantes <gabriela.cervantes.tellez@intel.com>
Date:   Tue Feb 27 02:48:54 2018 -0600

    tests: Add docker integration build test.
    
    This test the docker build command.
    
    Fixes #96
    
    Signed-off-by: Gabriela Cervantes <gabriela.cervantes.tellez@intel.com>
```

@GabyCT Can that file be removed? IIUC we no longer run kata -> docker integration tests...

**Comment by GabyCT** [comment]
yes @wainersm https://github.com/kata-containers/tests/pull/4224

**Comment by wainersm** [comment]
Can be closed as the PR#4224 removed the file.
